approach.md

```diff
@@ -0,0 +1,60 @@
+# OSEP proposed approach
+
+a) Document scope of analysis using STPA
+
+*In collaboration with other WGs e.g. Automotive?*
+
+* Assumed system context
+    - Hardware features, role of OS, boundaries of analysis, etc
+    - Doesn't have to be concrete / complex
+      - Specific to topic: start simple and elaborate later!
+* Losses
+    - OS-related outcomes that *may* violate a system's safety goals
+      - i.e. lead to harm in a safety-related system
+* Hazards
+    - OS-level system conditions that *may* lead to these losses
+* System-level constraints
+    - Criteria that must be satisfied to *prevent* or *mitigate* hazards
+      - May be a simple inversion of the hazard
+
+b) Identify and document system measures and mitigations for Linux
+
+*Out of scope for OSEP - LFSCS group?*
+
+* Provided by kernel features, compiler, hardware, etc
+* Find supporting evidence (design, tests, processes) for these
+* Identify responsibilities of components (kernel, compiler, etc)
+
+c) Perform hazard and risk analysis using STPA
```

Comment:
That might make more sense, but this will be an iterative process.

For b) I was thinking that we might include measures or mitigations that we believe Linux already provides, but it may be clearer to omit these on the first pass of analysis and then add them in for a second pass, to address gaps identified.